.storybook/stories/card.stories.mdx

```diff
@@ -1,105 +1,44 @@
-import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs';
-import { boolean, object, text, number, array } from "@storybook/addon-knobs";
-import { TabsWrapper, TabsTooltip, TabsWrapperShowTooltip } from "../misc";
+import {Meta, Story, Canvas, ArgsTable} from '@storybook/addon-docs';
 import CardContainer from '../../components/cards/CardContainer';
-import CardBody from '../../components/cards/CardBody';
-import CardHero from '../../components/cards/CardHero';
-import CardButton from '../../components/cards/CardButton';
-import FlexGridCol from '../../components/flexGrid/FlexGridCol';
-import FlexGridRow from '../../components/flexGrid/FlexGridRow';
+import CardBody from "../../components/cards/CardBody";
+import CardButton from "../../components/cards/CardButton";
+import CardHero from "../../components/cards/CardHero";
 
-export const imageLink = "./img/top-bowie-state-combined-image.svg";
-export const dummyText = "Pudding jelly beans tiramisu bear claw apple pie liquorice. Jelly beans macaroon wafer topping gummies. Shortbread tiramisu lollipop topping apple pie tart pie chocolate. Tart croissant cake danish wafer icing sugar plum.";
+<Meta title="Card/Default" component={CardContainer}
+ parameters={{options: {showAddonPanel: false}}} />
 
-<Meta
-    title="Card"
-    component={CardContainer}
-/>
+# Card
+```javascript
+import { CardContainer, CardHero, CardBody, CardButton } from 'data-transparency-ui';
+```
 
+Cards are UI Elements that contain content and actions about a single subject
+
+export const Template = (args) => <CardContainer {...args} />;
+
```

Comment:
default page - The prop tables for all of the components that make up a card is good.  I think we can the order of how we're displaying the variants and the prop tables.  I'm thinking...
1. Import statement
2. Description and how the component works with the variants (i'll write up a ticket to do this next sprint.  i'll include the exact text to add)
4. Container variants
5. Container props table
6. Hero variants
7. Hero props table
8. Card body props table
9. Card button props table